server/db.js

```diff
@@ -1,23 +1,56 @@
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/reviews', { useNewUrlParser: true, useFindAndModify: true });
-
-const db = mongoose.connection;
-
-db.on('error', console.error.bind(console, 'connection error:'));
-
-let reviewSchema = new mongoose.Schema({
-  productId: Number,
-  author: String,
-  rating: Number,
-  date: Date,
-  review: {
-    title: String,
-    review: String },
-  popularity: Number
+// const mongoose = require('mongoose');
+// mongoose.connect('mongodb://localhost/reviews', { useNewUrlParser: true, useFindAndModify: true });
+
+// const db = mongoose.connection;
+
+// db.on('error', console.error.bind(console, 'connection error:'));
+
+// let reviewSchema = new mongoose.Schema({
+//   productId: Number,
+//   author: String,
+//   rating: Number,
+//   date: Date,
+//   review: {
+//     title: String,
+//     review: String },
+//   popularity: Number
+// });
+
+// let Reviews = mongoose.model('Reviews', reviewSchema);
+
+// module.exports = {
+//   db, Reviews
+// };
+
+const Sequelize = require('sequelize');
+// const sequelize = new Sequelize('postgres://postgres@localhost:5432/reviews');
+const sequelize = new Sequelize('reviews', 'postgres', '', {
+  host: 'localhost',
+  dialect: 'postgres',
+  logging: false
 });
 
-let Reviews = mongoose.model('Reviews', reviewSchema);
+class Review extends Sequelize.Model{}
+
+Review.init({
+  productid: Sequelize.INTEGER,
+  author: Sequelize.STRING,
+  rating: Sequelize.INTEGER,
+  date: Sequelize.DATEONLY,
+  review: Sequelize.JSON,
+  popularity: Sequelize.INTEGER
+}, { sequelize, modelName: 'review' });
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connected to db!')
+  })
+  .catch(err => {
```

Comment:
This `.catch()` isn't actually handling your error - the error will be consoled automatically. If you aren't going to handle an error, you don't need `.catch()`!

Reply:
Took it out.